sagemaker-spark/pyspark_mnist/pyspark_mnist_kmeans.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.\n",
+    "\n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\").\n",
+    "You may not use this file except in compliance with the License.\n",
+    "A copy of the License is located at\n",
+    " \n",
+    "  http://aws.amazon.com/apache2.0/\n",
+    "\n",
+    "or in the \"license\" file accompanying this file. This file is distributed\n",
+    "on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either\n",
+    "express or implied. See the License for the specific language governing\n",
+    "permissions and limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SageMaker PySpark K-Means Clustering MNIST Example\n",
+    "\n",
+    "1. [Introduction](#Introduction)\n",
+    "2. [Loading the Data](#Loading-the-Data)\n",
+    "3. [Training and Hosting a Model](#Training-and-Hosting-a-Model)\n",
+    "4. [Inference](#Inference)\n",
+    "5. [More on SageMaker Spark](#More-on-SageMaker-Spark)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
```

Comment:
Can we add a comment like:
*This notebook was created and tested on an ml.m4.xlarge notebook instance.*
?

Reply:
Done